parser/v2/types.go

```diff
@@ -405,6 +405,17 @@ type WhitespaceTrailer interface {
 	Trailing() TrailingSpace
 }
 
+type HTMLExpression interface {
+	HTMLExpression() Expression
+}
+
+var (
+	_ HTMLExpression = Element{}
+	_ HTMLExpression = BoolConstantAttribute{}
+	_ HTMLExpression = BoolExpressionAttribute{}
+	_ HTMLExpression = ConstantAttribute{}
+)
```

Comment:
I added this with plans to eventually use the interface to determine when the HTML LSP can be enabled for intellisense.